com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -572,6 +572,13 @@ static GraphicsFormat MakeRenderTextureGraphicsFormat(bool isHdrEnabled, bool ne
             desc.bindMS = false;
             desc.useDynamicScale = camera.allowDynamicResolution;
 
+            if (SystemInfo.graphicsDeviceType == GraphicsDeviceType.Vulkan)
+            {
+                int requestedMsaaSamples = desc.msaaSamples;
+                if (requestedMsaaSamples == 2 && SystemInfo.GetRenderTextureSupportedMSAASampleCount(desc) == 1)
+                    desc.msaaSamples = 4;
+            }
```

Comment:
Fine for this PR but we should add a TODO here. Ideally URP should not have platform specific code anymore and this should be handled either in Core Unity